vulnfeeds/cpp/main.go

```diff
@@ -187,26 +195,30 @@ func CVEToOSV(CVE cves.CVEItem, repo, directory string) {
 	v.Affected = append(v.Affected, affected)
 
 	if len(v.Affected[0].Ranges) == 0 {
-		Logger.Infof("No affected versions detected for %s for %q", CVE.CVE.CVEDataMeta.ID, CPE.Product)
-		return
+		Logger.Warnf("No affected versions detected for %s for %q", CVE.CVE.CVEDataMeta.ID, CPE.Product)
```

Comment:
Same comment as above. We can just have a single log call from the caller instead of all this duplicaiton.

Reply:
Done.